sn_client/src/api.rs

```diff
@@ -74,9 +89,16 @@ impl Client {
 
         info!("Startup a client with peers {peers:?} and local {local:?} flag");
         info!("Starting Kad swarm in client mode...");
+        debug!("Starting Kad swarm in client mode.1..");
+        trace!("Starting Kad swarm in client mode..2.");
+
+        #[cfg(target_arch = "wasm32")]
+        let root_dir = PathBuf::from("dumb");
```

Comment:
same idea as above, having a unique function, say `get_root_dir` in seperate `target_arch_selection.rs` file
then here just needs to be:
`let root_dir = crate::target_arch_selection::get_root_dir();`

Reply:
I don't think its needed as this is the only location atm.